cli/cmd/auth.go

```diff
@@ -172,3 +214,32 @@ func authenticate(endPoint string, apiToken string) error {
 	rootCmd.SetArgs(args)
 	return rootCmd.Execute()
 }
+
+func smartKeptnCLIAuth() (string, error) {
+	keptnInstallations, err := keptnutils.GetKeptnManagedNamespace(false)
+	if err != nil {
+		return "", err
+	}
+
+	if len(keptnInstallations) > 1 {
```

Comment:
Can we add a parameter to the command to allow to specify one of the available Keptn installations? When used within a script (e.g. an integration test for this feature), the current approach would require a user input during the execution, and cause the script to stall at this point

Reply:
@bacherfl This is done, added a new flag `--ignore-namespace-listing`

Reply:
@ankitjain28may great for adding this flag. However, can I ask you a favor to rename it to `--skip-namespace-listing`. The reason is that another command has a similar flag we call `--skip...` (https://github.com/keptn/keptn/issues/2689)  and I would like to have both flags named the same way.

Reply:
@johannes-b Have made the changes as suggested.